cmd/manager/main.go

```diff
@@ -232,6 +232,11 @@ func printConfig(cfg *configuration.Registry) {
 		logWithValues = logWithValues.WithValues("key", key, "value", value)
 	}
 	logWithValues.Info("Registration Service configuration variables:")
+
+	for key, value := range cfg.GetAllHostOperatorParameters() {
+		logWithValues = logWithValues.WithValues("key", key, "value", value)
+	}
+	logWithValues.Info("Host Operator configuration variables:")
```

Comment:
we may not want to do that if in the future we have secrets (credentials, etc.) in the config.

Reply:
Good point. So we have two ways to avoid that:
1. filter out the variables that contain the secrets here - where we print the vars in the logs
2. call directly the functions and collect the list of the keys and values

The proper way is the second solution, but we cannot do it for the registration-service :-/ 
Technically, the secrets for the registration-service should not have been stored within the host-operator variable, so we should be safe in that case.
Let me try to rewrite it so we gather all vars and keys of the host-operator directly by calling the functions.

Reply:
done in 26f377e

Reply:
but why do we need to print the key/values, especially if there's a risk of leaking credentials?

Reply:
I mean, if we can't avoid such a leak, shouldn't we just avoid dumping all key/value pairs in the logs?

Reply:
The reason why I started printing out the variables was that I found out very difficult to figure out what was going on when the deployment of the registration service failed. 
I thought that it would be good to know which parameters are used for the reg-service deployment - mainly when we have different configuration values/parameters for different environments. I need to say that this printing was very helpful a few times and it saved me a lot of time.

Reply:
ok, let's address this issue in a separate PR, then